osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -166,25 +157,30 @@ private double computeSpeedValue()
 
         private double computeAccuracyValue()
         {
-            // This percentage only considers HitCircles of any value - in this part of the calculation we focus on hitting the timing hit window
-            double betterAccuracyPercentage;
             int amountHitObjectsWithAccuracy = Attributes.HitCircleCount;
 
-            if (amountHitObjectsWithAccuracy > 0)
-                betterAccuracyPercentage = ((countGreat - (totalHits - amountHitObjectsWithAccuracy)) * 6 + countOk * 2 + countMeh) / (double)(amountHitObjectsWithAccuracy * 6);
-            else
-                betterAccuracyPercentage = 0;
+            // This section should be documented by Tr3, but effectively we're calculating the exact same way as before, but
+            // we calculate a variance based on the object count and # of 50s, 100s, etc. This prevents us from having cases
+            // where an SS on lower OD is actually worth more than a 95% on OD11, even though the OD11 requires a greater
+            // window of precision.
+
+            double p100 = (double)countOk / amountHitObjectsWithAccuracy;
+            double p50 = (double)countMeh / amountHitObjectsWithAccuracy;
+            double pm = (double)countMiss / amountHitObjectsWithAccuracy;
+            double p300 = 1.0 - pm - p100 - p50;
 
-            // It is possible to reach a negative accuracy with this formula. Cap it at zero - zero points
-            if (betterAccuracyPercentage < 0)
-                betterAccuracyPercentage = 0;
+            double m300 = 79.5 - 6.0 * Attributes.OverallDifficulty;
+            double m100 = 139.5 - 8.0 * Attributes.OverallDifficulty;
+            double m50 = 199.5 - 10.0 * Attributes.OverallDifficulty;
+            double acc = p300 + 1.0 / 3.0 * p100 + 1.0 / 6.0 * p50;
```

Comment:
var never used

Reply:
Just as a heads-up, these kinds of issues are already pointed out in our style rules (and still need to be applied to this PR).

https://ci.appveyor.com/project/peppy/osu/builds/39360000